### ISSUE TITLE
Symbol’s function definition is void: message!

### ISSUE THREAD

**Issue by phoenixanimations** [jsmestad/doom-todo-ivy#1]
Hey, downloaded the package and when trying to run `doom/ivy-tasks` I get `Symbol’s function definition is void: message!` when I take out:

```elisp
(message! (red "Error matching task in file: (%s) %s"
                                      (error-message-string ex)
                                      (car (split-string x ":"))))
```
It works as expected. 

I believe their might be another package being used, I tested this on emacs 26.1, doing `emacs -Q`, `(package refresh)` and copying and running doom-todo-ivy.el in scratch.

**Comment by jueqingsizhe66** [comment]
`M-x doom/ivy-tasks` -----------> **max:Wrong type argument:number or marker-p,nil**

**Comment by jsmestad** [comment]
@phoenixanimations ah that is a hold-over from doom-emacs. I can get that fixed up.

**Comment by jsmestad** [comment]
@phoenixanimations give the latest version a shot

**Comment by phoenixanimations** [comment]
Just downloaded it from github, unfortunately I think there is one last function from doom-emacs, which is `red`, in the same line of code.

**Comment by jsmestad** [comment]
@phoenixanimations would you mind tossing a quick PR together with the fix? I can't get that line to trigger on my setup to diagnose the error.

**Comment by phoenixanimations** [comment]
Pull request sent, I believe that should solve the issue.

**Comment by jsmestad** [comment]
@phoenixanimations merged #2 :)

**Comment by phoenixanimations** [comment]
Yay! Works great now, thank you for porting this :).
